### DISCUSSION TITLE
Snippets

### DISCUSSION THREAD

**Discussion by tjdevries** [tjdevries/rofl.nvim#6] [General]
I want to work on some stuff related to snippets. At this point, I'd just make something that is by default LSP snippet compatible, because I'd like to have a better experience with that and then take some of those lessons and implement in Neovim core perhaps (but that's somewhat unrelated to this plugin).

I don't necessarily have to implement everything for snippets, but it's good practice for making it able to be snippet compatible.

**Comment by OldManFroggy** [comment]
curious to see what you come up with, to-date I have never really got into the habit of using snippets.
